src/data/questions.json

```diff
@@ -10760,6 +10366,36 @@
           "frequency": 2
         }
       ]
+    },
```

Comment:
Please include only the changes relevant to your PR - we should avoid formatting other sections of this file. Also, the `companies` section is incorrect - we can leave it as an empty JSON array for now and it will be populated when [update_questions.py](https://github.com/seanprashad/leetcode-patterns/blob/main/cron/update_questions.py) runs next Sunday